osquery/tables/events/windows/windows_events.cpp

```diff
@@ -42,7 +42,12 @@ class WindowsEventSubscriber
  public:
   Status init() override {
     auto wc = createSubscriptionContext();
-    for (const auto& chan : osquery::split(FLAGS_windows_event_channels, ",")) {
+    for (auto& chan : osquery::split(FLAGS_windows_event_channels, ",")) {
+      // We remove quotes if they exist
+      auto quoteLoc = chan.find("\"");
+      if (quoteLoc != std::string::npos) {
+        chan.erase(quoteLoc);
```

Comment:
Good call. I think those all might break it. The objective is to get rid of all quotes. Thanks dude!